src/Coders.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <string>
+#include <iostream>
+
+
+class Coders
+{
+private:
+    std::string userName;
+    std::string discordNickname;
+    std::string githubLink;
+    std::string firecodeLink;
+public: 
+    enum class Group {weekends, evenings};
+    Group SetGroup;
+    Coders(std::string userName_,
+           std::string discordNickname_, 
+           std::string githubLink_,
+           std::string firecodeLink_,
+           Group SetGroup_);
```

Comment:
I would suggest the opposite convention - parameters without underscores, but member fields with underscores.